contributors/design-proposals/storage/csi-migration.md

```diff
@@ -35,12 +35,28 @@ lower cost of development as we only have to maintain one version of each
 plugin, as well as ease the transition to CSI when we are able to deprecate the
 internal APIs.
 
+The migration from in-tree plugins to CSI plugins will involve the following phases:
+
+Phase 1: In-tree plugins are probed but the in-tree plugin code is largely skipped
+for operations on persistent volumes in favor of CSI plugins that supersede the
+in-tree plugins. In case a node does not have a CSI plugin (that supersedes a
```

Comment:
I'm not sure about the introduction of this new terminology "supersceding" the plugin. Lets continue to call it "translated/migrated" so we don't call the same thing by multiple names